buildSrc/src/main/kotlin/androidx/build/AndroidXPlaygroundRootPlugin.kt

```diff
@@ -157,6 +157,10 @@ class AndroidXPlaygroundRootPlugin : Plugin<Project> {
         google()
         mavenCentral()
         jcenter()
+        // TODO temporary until AndroidX merge
+        maven {
+            it.url = URI("https://jitpack.io")
```

Comment:
Probably need to remove this?

Reply:
good catch.
I also need to move away from its snapshot to an actual release but that is breaking the build right now, will take a look but may need to wait for another release from kotlin compile testing.